src/styles/views/Rules.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/theme';
+
+.rules {
+
+    &.main-container{
+         margin-top: 2em;
+         min-height: 300px;
+         @include center;
+         flex-direction: column;
+         margin-bottom: 20px;
+     }
+    &.card-container{
+         background-color: $accent;
+         display: block;
+         @include center;
+
+     }
+
+    &.info-container{
+      color: $background;
+      font-size: $fontSizeSmall;
+      margin-bottom: 50px;
+     }
+
+    &.smallTitle{
+       font-weight: bold;
+       color: $background;
+       position: relative;
+       font-size: $fontSizeMediumSmall;
+      margin-bottom: -10px;
```

Comment:
-10px is a bit much, would rather change to -5px

Reply:
done